+bert/+tokenizer/BERTTokenizer.m

```diff
@@ -99,7 +99,7 @@
             this.MaskCode = this.FullTokenizer.encode(this.MaskToken);
         end
         
-        function tokens = tokenize(this,text_a,text_b)
+        function [tokens,normalWords] = tokenize(this,text_a,text_b)
```

Comment:
I wonder if we should make the 2nd output a struct in case we want to return more information later (and not keep extending this to a 3 or more output function).

I don't have a good example of what extra information would be returned currently. I could imagine extending `BERTTokenizer` to do padding and return the padding masks, and to return the "type IDs" (i.e. integer labels for if a token is from `text_a` or `text_b`).

Reply:
True. And it is probably not that useful either to just return a logical saying where words start - we probably want the initial tokenization results instead, as there is no reconstructing from `[UNK]`.

For the task I am working on, the other input we were talking about is probably going to be more useful than this. If we go that route, I'm going to drop this PR and orphan or delete this branch, yes?

Reply:
Ok, sounds fine to me, we should discuss the details. I'm happy to leave this open until then.